treematcher.py

```diff
@@ -1,286 +1,178 @@
-from itertools import permutations
-from ete3 import NCBITaxa
-import six
+#system modules
 import re
-import ast
+from itertools import permutations
 
+# third party modules
+import ast
+import six
 from ete3 import PhyloTree, Tree, NCBITaxa
 
+# internal modules
+# ...
 
-class TreePattern(Tree):
-    """
-        Class for presenting a tree of constrained patterns. A way to represent patterns to search for within trees.
-        :param args: Pattern to be searched.
-        :param kargs: Custom functions and arguments passed to Tree Class.
-    """
-    _syntax_tuples = [
```

Comment:
these tuples are no longer necessary using, only translation from @ to __target_node